mathics/builtin/lists.py

```diff
@@ -617,7 +618,7 @@ def list_boxes(items, f, open=None, close=None):
         sep = ","
     result = riffle(result, String(sep))
     if len(items) > 1:
-        result = Expression("RowBox", Expression(SymbolList, *result))
+        result = Expression(SymbolRowBox, Expression(SymbolList, *result))
```

Comment:
With the current changes, this could be directly ``RowBox(*results)`` which avoids an evaluation step.

Reply:
Good! - Trying it now...

Reply:
No I tried it and I get a long ugly traceback when running: 

```
python -x mathics/docpipeline.py -s 'Formatting Output'
```

If you can get this to work, I'd be grateful.

Reply:
#313 merge this with #311, which fixes this.